tools/js-sdk-release-tools/src/test/changelogGenerator.test.ts

```diff
@@ -92,3 +92,38 @@ test("Modular -> Modular: Change Op", async () => {
         "Removed operation DataProductsCatalogsOperations.listByResourceGroup"
     );
 });
+
+test("HLC -> HLC: Operation Group Add/Remove/ChangeSig", async () => {
+    const oldViewPath = path.join(
+        __dirname,
+        "testCases/operationGroups.4.old.hlc.api.md"
+    );
+    const newViewPath = path.join(
+        __dirname,
+        "testCases/operationGroups.4.new.hlc.api.md"
+    );
+    const changelog = await extractExportAndGenerateChangelog(
+        oldViewPath,
+        newViewPath,
+        SDKType.HighLevelClient,
```

Comment:
has the modular vs HLC change log merged? I am curious if there's any test cases for rename in modular vs HLC scenario?